python/ray/tune/examples/tune_cifar10_gluon.py

```diff
@@ -194,11 +194,7 @@ def test():
         sched = FIFOScheduler()
     elif args.scheduler == "asynchyperband":
         sched = AsyncHyperBandScheduler(
-            time_attr="training_iteration",
-            metric="mean_loss",
-            mode="min",
-            max_t=400,
-            grace_period=60)
+            metric="mean_loss", mode="min", max_t=400, grace_period=60)
```

Comment:
We could also pass metric/mode to tune.run here

Reply:
will do this in followup!